pages/home/tasks.js

```diff
@@ -29,26 +31,10 @@ export default function Home() {
       setShouldReload(false);
     };
     if (shouldReload) {
-      getProducts();
+      getTasks();
     }
   }, [shouldReload]);
 
-  // delete function for the future
-
-  // async function handleDelete(id) {
-  //   try {
-  //     const response = await fetch(`/api/tasks/${id}`, {method: "DELETE"});
-  //     if (response.ok) {
-  //       console.log("Product deleted!");
-  //     } else {
-  //       throw new Error(`Fetch fehlgeschlagen mit Status: ${response.status}`);
-  //     }
-  //   } catch (error) {
-  //     alert(error.message);
-  //   }
-  //   setShouldReload(true);
-  // }
-
   return (
     <div>
```

Comment:
Use a `fraction` element instead of a `div`
```suggestion
    <>
```